app/components/gh-file-uploader.js

```diff
@@ -74,6 +76,20 @@ export default Component.extend({
         }
     },
 
+    didReceiveAttrs() {
+        this._super(...arguments);
+        let accept = this.get('accept');
+        let extensions = this.get('extensions');
+
+        if (!isBlank(accept) && !isEmberArray(accept)) {
+            this.set('accept', accept.split(','));
```

Comment:
I tried doing that and I got a lot of undefined errors - will update it to not use two-way binding but it didn't seem to like me just setting properties on the component.